Makefile

```diff
@@ -139,8 +139,11 @@ docker-otelcontribcol:
 
 .PHONY: otelcontribcol
 otelcontribcol:
-	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_$(GOOS)_$(GOARCH)$(EXTENSION) $(BUILD_INFO) ./cmd/otelcontribcol
+	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_$(GOOS)_$(GOARCH)$(EXTENSION) \
+		$(BUILD_INFO) ./cmd/otelcontribcol
 
+	GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelcontribcol_exp_$(GOOS)_$(GOARCH)$(EXTENSION) \
```

Comment:
Naming being one of the 2 hard problems in computer science, I'd avoid `exp` for `experimental` (expired, experience, export, ...). How about alpha vs. stable? Or "unstable", a la Debian?

Also, would be nice to add an entry about this in the readme, explaining what's the rule for determining what should go where.

Reply:
Renamed everything to "unstable" instead of "experimental".